### PR TITLE
docs(manual): add documentation for set language hint feature

### LINKED ISSUE #66: [2.13] Explain "Set Language Hint" feature

### PR DESCRIPTION
**Pull request by avinashbest**
**Explained the Automatic Language Selection feature/Set Language Hint feature**

Fixes #66

### CODE REVIEW

File: manual.asc [hunk 1]
```diff
@@ -1052,6 +1052,13 @@ To stop receiving notifications go to Deck options > Reminders and unmark the ch
 
 Notifications only work for top level decks. Please let us know if you want us to add notifications for subdecks too.
 
+[[setlanguagehint]]
+=== Automatic Language Selection
+AnkiDroid will help you in Automatic Language Selection feature from AnkiDroid 2.13. This feature allows you to define a default language for a field in a note type.
```
Comment (mikehardy) [inline manual.asc:1057]:
```suggestion
AnkiDroid has an in Automatic Language Selection feature in the Note Editor starting with AnkiDroid 2.13. This feature allows you to define a default language to be used in your keyboard for a field in a note type.
```

File: manual.asc [hunk 2]
```diff
@@ -1052,6 +1052,13 @@ To stop receiving notifications go to Deck options > Reminders and unmark the ch
...
+
+For e.g. You have Russian and English translation fields. So, the layout (in the apps SwiftKey and Swype as most popular, for example) switches to Russian in the first field and back to English in the second automatically when the fields get focus. https://youtu.be/JrxDjTrRhBE[Checkout video reference for demonstration]
```
Comment (mikehardy) [inline manual.asc:1059]:
```suggestion
For example, if you have Russian and English note fields, the keyboard layout (in keyboards like SwiftKey and Swype) will switch to Russian in the first field and back to English in the second automatically when the fields get focus. https://youtu.be/JrxDjTrRhBE[Checkout video reference for demonstration]
```

**Review by mikehardy — APPROVED** [review]
Nice! Thanks for getting to this, it had been on our list for a long time, and while it is easy to polish a few words, actually starting the task obviously had not happened :-)

So I edited the grammar a little and I think it is good to go now

I also altered the PR title, it is not a requirement on this project, but in other projects (notably: in lots of javascript projects like the react-native ecosystem and the Angular ecosystem) they use "conventional commits" which are PR titles and commit messages that follow a specific format, and then releases are easy to automatically generate because the commit messages relate well to semantic versioning

https://www.conventionalcommits.org/en/v1.0.0/
https://semver.org/

File: manual.asc [hunk 3]
```diff
@@ -1052,6 +1052,13 @@ To stop receiving notifications go to Deck options > Reminders and unmark the ch
 
 Notifications only work for top level decks. Please let us know if you want us to add notifications for subdecks too.
 
+[[setlanguagehint]]
+=== Automatic Language Selection
+AnkiDroid has an in Automatic Language Selection feature in the Note Editor starting with AnkiDroid 2.13. This feature allows you to define a default language to be used in your keyboard for a field in a note type.
+
+For example, if you have Russian and English note fields, the keyboard layout (in keyboards like SwiftKey and Swype) will switch to Russian in the first field and back to English in the second automatically when the fields get focus. https://youtu.be/JrxDjTrRhBE[Checkout video reference for demonstration]
```
Comment (david-allison-1) [inline manual.asc:1059]: > in keyboards like SwiftKey and Swype

The last time that I checked, SwiftKey doesn't support `setImeHintLocales`

https://support.swiftkey.com/hc/en-us/community/posts/360055542372-SwiftKey-does-not-switch-setImeHintLocales- (2019)

Swype's been pulled from the Play Store.

Comment (mikehardy) [reply manual.asc:1059]: Haha - that's amusing, both examples as fails. I guess the lesson is: do not give examples by name in documentation, simply state the requirement they must meet ("that support setImeHintLocales", the standard keyboard language hint system in Android")

Comment (mikehardy) [reply manual.asc:1059]: @avinashbest can you propose a quick follow-up PR that removes the named keyboard apps and just states they need to support the setImeHintLocales Android API ?

Comment (avinashbest) [reply manual.asc:1059]: Sure, Why not

**Pull request merged**